### ISSUE TITLE
Resettable default does not appear to be true

### ISSUE THREAD

**Issue by mattwd7** [splitrb/split#598]
**Describe the bug**
I have added the split gem to my Rails project and created an experiment in a YAML file that is loaded in an initializer `split.rb`. After a user converts and the `#ab_finished` helper is called, the conversion is recorded and visible from the Split dashboard. However, future participations/conversions are not recorded for the same user as expected.

**To Reproduce**
1. Create an experiment in your split initializer without setting `resettable: true`
```
join_button_color:
  alternatives:
    - name: 'control'
      percent: 50
    - name: 'blue'
      percent: 50
```

2. Consult the experiment in a view partial to determine how to class your button
```
button_class = (ab_test('join_button_color') == 'blue') ?
    'btn-outline-primary' :
    'btn-outline-success'
```

3. Call `ab_finished` at the controller action where the button links to. Afterward, the dashboard should show 1 participant and 1 completed for the alternative selected.
`ab_finished(:join_button_color)`

4. After converting the user, return to a page with the button partial. Consulting the dashboard shows that the user's experiment was not reset and the participant count is unchanged.

**Additional context**
After a bit of digging, I wonder if the culprit is `Split::Expeirment#set_alternatives_and_options` called within `#initialize`. In this method, `self.resettable = options[:resettable]` is assigning `nil` or `""`, which evaluates to `false` in the setter. (Or I'm just doing something wrong – help!!)

**Comment by andrehjr** [comment]
Hi @mattwd7

I can confirm that this is a bug. It only happens when loading the configuration from the split.rb initializer. I'll address this before releasing the next version, which I expect to be some time next week.

**Comment by mattwd7** [comment]
Thanks for the reply @andrehjr! I'll try poking around myself

**Comment by mattwd7** [comment]
I've added a PR: https://github.com/splitrb/split/pull/599

**Comment by mattwd7** [comment]
Any update on this? No comments on PR and would hate for it to become stale

**Comment by andrehjr** [comment]
Hey, @mattwd7 sorry, this actually fell under my radar. I'll take a look into it soon, it won't become stale. ✨

**Comment by andrehjr** [comment]
Closed by #599
